src/include/sof/audio/pipeline.h

```diff
@@ -86,8 +86,12 @@ extern struct tr_ctx pipe_tr;
 #define PPL_DIR_DOWNSTREAM	0
 #define PPL_DIR_UPSTREAM	1
 
+#ifdef MAILBOX_STREAM_SIZE
```

Comment:
Separate commit with a comment here describing why this is not defined for some platforms.